spot_rl_experiments/spot_rl/utils/skill_executor.py

```diff
@@ -26,6 +26,7 @@ class SpotRosSkillExecutor:
     def __init__(self, spotskillmanager):
         self.spotskillmanager: SpotSkillManager = spotskillmanager
         self._cur_skill_name_input = None
+        self._pre_in_dock = False
```

Comment:
```suggestion
        self._is_robot_on_dock= False
```

Replace _pre_in_dock with explicit names?

Reply:
done!